kord-extensions/src/main/resources/translations/kordex/strings.properties

```diff
@@ -65,6 +65,26 @@ checks.topRoleHigher.failed=Must have a top role higher than: **{0}**
 checks.topRoleLower.failed=Must have a top role lower than: **{0}**
 checks.topRoleHigherOrEqual.failed=Must have a top role of **{0}**, or a higher top role
 checks.topRoleLowerOrEqual.failed=Must have a top role of **{0}**, or a lower top role
+cooldown.notifier.commandUser=You may use `{1}` again {0}.
+cooldown.notifier.commandUserChannel=You may use `{1}` in {2} again {0}.
+cooldown.notifier.commandUserGuild=You may use `{1}` in this server again {0}.
+cooldown.notifier.globalUser=You may use commands again {0}.
+cooldown.notifier.globalUserChannel=You may use commands in {1} again {0}.
+cooldown.notifier.globalUserGuild=You may use commands in this server again {0}.
+cooldown.notifier.global=Someone may use commands again {0}.
+cooldown.notifier.globalChannel=Someone may use commands in {1} again {0}.
+cooldown.notifier.globalGuild=Someone may use commands in this server again {0}.
```

Comment:
These should be "Anyone" probably, not "Someone". However, I'm not sure whether any of these above strings are particularly useful - I don't think we should be telling people when their cooldowns expire, that could get incredibly spammy.